products/run.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+nameko run --config config.yml products.service
```

Comment:
Unfortunately this service will die because it will try to start before Rabbit can accept connections. There is a workaround implemented here that can just be copied: https://github.com/nameko/nameko-examples/blob/master/orders/run.sh#L5
If you have a better idea how to do it please shout.